py/ngff_zarr/v06/zarr_metadata.py

```
@@ -0,0 +1,425 @@
+            coordinateTransformations=None,
+        )
+
+        return metadata
+    
+    @classmethod
+    def _from_zarr_attrs(
+        cls,
+        root_attrs: dict,
+        store: StoreLike,
+        validate: bool = False,
+        ) -> tuple["Metadata", list["NgffImage"]]:
```

Ruff (0.15.12)

[error] Undefined name `NgffImage`

(F821)